Lib/test/datetimetester.py

```diff
@@ -1520,6 +1520,13 @@ def test_replace(self):
         base = cls(2000, 2, 29)
         self.assertRaises(ValueError, base.replace, year=2001)
 
+    def test_subclass_replace(self):
+        class _DateSubclass(self.theclass):
```

Comment:
Please remove "_" prefix in class names. These classes are only defined in the scope of the test, no need to declare them as "private" with the "_" prefix.